ui/css/layout/_typography.scss

```diff
@@ -94,21 +94,12 @@ h3,
   line-height: 1.5rem;
 }
 
-.document-container {
-  h4,
-  .h4 {
-    color: $color-ochre;
```

Comment:
According to my `grep`, this might be the only place `$color-ochre` is used outside of its definition in `_colors.css`--should we remove its definition?

Reply:
Good question; we have in the past, but let's verify with @rtwell that the color won't be used somewhere else soon.

Reply:
@rtwell requests we keep it